.fluidlintallrc.json

```diff
@@ -0,0 +1,36 @@
+{
+    "eslint": {
+        "js": {
+            "excludes": ["tests/fixtures/js/bad.js"]
+        },
+        "json": {
```

Comment:
What is the different between this check and the `jsonlint` check?

Reply:
`jsonlint` mainly checks validity, and allows us to report better position information about the errors than `JSON.parse`.

`eslint.json` (which I just added docs for) wraps JSON(5) content in a wrapper, as though it were part of a Javascript file, as in:

```
"use strict";
// eslint-disable-next-line no-unused-vars
var foo = <JSON(5) payload>;

```

This wrapped content is run through our standard ESLint Javascript, with the position of the error updated to reflect the position in the JSON(5) content.  In this way we can catch a much wider range of issues with JSON(5) content.

Reply:
The mechanism is clear, but the function still isn't quite. What issues do we expect in practice to be caught by the eslint test that we didn't get via jsonlint? Indentation or what?

Reply:
Trailing commas are a good example, as is indentation.

Reply:
Bizarre if jsonlint doesn't pick up trailing commas, since these are invalid JSON?

Reply:
You're right.  I went back to the [universal pull where we introduced linting](https://github.com/GPII/universal/pull/623) to refresh my memory.  Seems like the check basically covers indentation, trailing *spaces* after values, and badly formatted comments in JSON5 files.